ref_arch/multi_tenant.rst

```diff
@@ -0,0 +1,393 @@
+.. highlight:: postgresql
+
+Multi-tenant Applications
+#########################
+
+.. contents::
+
+*Estimated read time: 30 minutes*
+
+Many companies with web applications cater not to end users, but to other businesses with customers of their own. When many clients with similar needs are expected, it makes sense to run a single instance of the application to handle all the clients.
+
+A software-as-a-service (SaaS) provider, for example, can run one instance of its application on one instance of a database and provide web access to multiple customers. In such a scenario, each tenant's data is isolated and remains invisible to other tenants. This is efficient in two ways. First application improvements apply to all clients. Second, sharing a database between tenants uses hardware efficiently.
```

Comment:
"In such a scenario, each tenant's data is isolated and remains invisible to other tenants. " -> "In such a scenario, a single software system serves multiple customers, each with their own isolated view of their own data in that system"